blackjackTrainer-backend/src/main/java/com/blackjacktrainer/blackjacktrainerbackend/User/DataLayer/Client.java

```diff
@@ -21,4 +21,5 @@ public class Client {
     public String email;
     public String phoneNumber;
     public String address;
+    public double height;
```

Comment:
Why did you add this? we dont need height for blackjack players.

Reply:
Ummmm actually you need height.